src/Ocelot.Provider.Polly/v7/PollyPoliciesDelegatingHandler.cs

```diff
@@ -2,12 +2,13 @@
 using Microsoft.Extensions.DependencyInjection;
 using Ocelot.Configuration;
 using Ocelot.Logging;
-using Ocelot.Provider.Polly.Interfaces;
+using Ocelot.Provider.Polly.v7;
 using Polly.CircuitBreaker;
 using System.Diagnostics;
 
 namespace Ocelot.Provider.Polly;
```

Comment:
namespace is wrong

Reply:
Btw, not your fault, but _logger can be removed.

Reply:
Agree! Namespace was not adjusted. But Visual Studio offers to adjust namespaces always while doing file/folder operation.

Reply:
Actually it's not resolved! Unresolving...

Reply:
Fixed in https://github.com/ThreeMammals/Ocelot/pull/1914/commits/f4aca4e0b749e5d6d185aa1e4d7866cde3561fc7